schemas/schema_helpers.go

```diff
@@ -178,6 +178,33 @@ func diffSuppressInterfaceListOrder(mapKey string) schema.SchemaDiffSuppressFunc
 	}
 }
 
+// diffSuppressAppInterfaceListOrder checks whether we have the newly created AppInterface
+// or not. It suppress any differences after an AppInstance is created.
+// AppInterfaces are never updated after they are created
+func diffSuppressAppInterfaceListOrder(mapKey string) schema.SchemaDiffSuppressFunc {
+	return func(key, oldValue, newValue string, d *schema.ResourceData) bool {
+		oldData, newData := d.GetChange(mapKey)
+		if newData == nil && oldData == nil {
+			return true
+		}
+
+		if oldData == nil {
+			return false
+		}
+		if newData == nil {
+			return false
+		}
+
+		o := oldData.([]interface{})
+		n := newData.([]interface{})
+		if len(o) != len(n) {
+			return false
+		}
+
+		return true
```

Comment:
Here we would just ignore any change the user made in their config to existing interfaces, correct? So no change would happen. I think there is no way to address this scenario, we are assuming the user knows that changes will be ignored or they will contact support reporting a bug.
Do you know if the interface behavior is documented anywhere?

Reply:
Yes, it will ignore any changes to app interfaces after app instance creation. It has the same behaviour as the backend does handling the API request on app instance update. 

> Do you know if the interface behavior is documented anywhere?
no, i didn't meet any documentation about that